src/DataMigration/custom/Cmdlets/Register-AzDataMigrationIntegrationRuntime.ps1

```diff
@@ -0,0 +1,59 @@
+function Register-AzDataMigrationIntegrationRuntime 
```

Comment:
Please add copyright. Refer to Get-AzDataMigrationAssessment.ps1

Reply:
Updated